apis/r/configure

```diff
@@ -2,20 +2,18 @@
 
 ## This allow for standard CRAN override preference for both a settable R_HOME
 ## with fallback to query R in $PATH for the value it has so it works both
-## explicitly or implicitly from the running R instance
+## explicitly, implicitly from the running R instance or by pointing at alternate
+## build when multiple R versions are installed (as CRAN does and some users do)
 : ${R_HOME=`R RHOME`}
 if test -z "${R_HOME}"; then
     echo Could not determine R_HOME.
     exit 1
 fi
 
-## look for tiledb core library and either use system library or download build
-have_tiledb="false"
-
-## check for pkg-config and use it to inquire about tiledb build options
+## Check for pkg-config and use it to inquire about tiledb and tiledbsoma build options
 pkg-config --version >/dev/null 2>&1
 if [ $? -eq 0 ]; then
-    pkg-config --exists tiledb
+    pkg-config --exists tiledb tiledbsoma
     if [ $? -eq 0 ]; then
         pkgcflags=`pkg-config --cflags tiledb tiledbsoma`
```

Comment:
I think these need to be checked independently. It's quite possible someone will have local/edited tiledbsoma C++ source but not local/edited core source.

Reply:
Also separate/independent `sed` statements for `tiledb_include`/`tiledb_libs`, and `tiledbsoma_include`/`tiledbsoma_libs`

Reply:
No, this is very much on purpose.